storagenode/storagenodedb/storagenodedbtest/satellites_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedbtest_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode"
+	"storj.io/storj/storagenode/satellites"
+	"storj.io/storj/storagenode/storagenodedb/storagenodedbtest"
+)
+
+// initiate graceful exit doesn't explode
+func TestInitiateGracefulExitDoesNotExplode(t *testing.T) {
```

Comment:
After a `Get...` function is added, we should improve all these tests by using it to validate that the data is what we expect.